core/controllers/reader.py

```diff
@@ -79,6 +79,48 @@ def _does_exploration_exist(exploration_id, version, collection_id):
 class ExplorationEmbedPage(base.BaseHandler):
     """Page describing a single embedded exploration."""
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'exploration_id': {
+            'schema': {
+                'type': 'basestring',
+                'validators': [{
+                    'id': 'is_regex_matched',
+                    'regex_pattern': constants.ENTITY_ID_REGEX
+                }]
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'v': {
+                'schema': {
+                    'type': 'int',
+                    'validators': [{
+                        'id': 'is_at_least',
+                        'min_value': 1
+                    }]
+                },
+                'default_value': None
+            },
+            'collection_id': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'is_regex_matched',
+                        'regex_pattern': constants.ENTITY_ID_REGEX
+                    }]
+                },
+                'default_value': None
```

Comment:
Why is the `default_value` `None`?

Reply:
Because the comment above said that it was an optional argument and the wiki stated that the `default_value` of optional arguments should map to null

Reply:
Which comment above? If some argument is optional should be evaluated against frontend.

Reply:
<img width="507" alt="Screenshot 2022-01-03 at 11 09 32 PM" src="https://user-images.githubusercontent.com/63154588/147961819-1699dc5b-be22-4961-bb61-7ce22015a8fe.png">
That was the comment that I saw

Reply:
Okay, Noted

Reply:
Okay, makes sense.